public/app/plugins/datasource/cloudwatch/query_parameter_ctrl.ts

```diff
@@ -1,12 +1,11 @@
 import angular from 'angular';
 import coreModule from 'app/core/core_module';
 import _ from 'lodash';
-import { TemplateSrv } from 'app/features/templating/template_srv';
-import DatasourceSrv from 'app/features/plugins/datasource_srv';
+import { TemplateSrv } from '@grafana/runtime';
 
 export class CloudWatchQueryParameterCtrl {
   /** @ngInject */
-  constructor($scope: any, templateSrv: TemplateSrv, uiSegmentSrv: any, datasourceSrv: DatasourceSrv) {
```

Comment:
Is this an intentional change?

Reply:
Yes, since the class doesn't actually make use of datasourceSrv I thought it could just be removed

Reply:
Ok, sounds good!